spog/ui/src/pages/scanner/mod.rs

```diff
@@ -48,7 +48,7 @@ fn is_supported_package(purl: &str) -> bool {
             package.ty() == "maven"
                 || package.ty() == "gradle"
                 || package.ty() == "npm"
-                || package.ty() == "gomodules"
+                || package.ty() == "golang"
                 || package.ty() == "pip"
```

Comment:
I believe the correct scheme for Python is `pypi`: https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#pypi

Reply:
Thank you very much @ctron your are right!